apps/beeswax/src/beeswax/server/hive_server2_lib_tests.py

```diff
@@ -64,6 +64,7 @@ def test_open_session(self):
     )
 
     with patch('beeswax.server.hive_server2_lib.thrift_util.get_client') as get_client:
+      original_secret = original_guid = b'1'
```

Comment:
Nit: maybe replace it by one of your full string? (more realistic on the issue)

Reply:
Done